lib/PrefsFeed.jsm

```diff
@@ -46,9 +46,8 @@ this.PrefsFeed = class PrefsFeed {
   }
 
   onPrefChanged(name, value) {
-    const prefItem = this._prefMap.get(name);
-    if (prefItem) {
-      this.store.dispatch(ac[prefItem.skipBroadcast ? "OnlyToMain" : "BroadcastToContent"]({type: at.PREF_CHANGED, data: {name, value}}));
+    if (this._prefMap.has(name)) {
```

Comment:
How was this breaking? The PREFS_CONFIG we pass in should be a mapping from strings to objects, so even if a pref wanted a false value, the map get would result in `{ title: "foo", value: false }` ???

Is this something we need to uplift too?

Reply:
oh hmm. interesting... but the tests fail without this change and pass when i apply it... 😕

Reply:
Oh.. I see. Specially for prefs defined in firefox.js, the prefMap gets a plain value instead of an object.

https://github.com/mozilla/activity-stream/blob/b4ec95c93d493ffd40d8c0c6778fe1c934b0b0e1/lib/PrefsFeed.jsm#L89-L101